openfeedback/src/main/java/io/openfeedback/android/mappers/FirestoreToModelMappers.kt

```diff
@@ -0,0 +1,16 @@
+package io.openfeedback.android.mappers
+
+import com.google.firebase.Timestamp
+import io.openfeedback.android.model.Comment
+
+internal fun Map<String, *>.convertToModel(
+    id: String,
+    voteItemId: String
+): Comment = Comment(
+    id = id,
+    voteItemId = voteItemId,
+    text = this["text"] as String,
+    plus = this["plus"] as Long,
+    createdAt = this["createdAt"] as Timestamp,
```

Comment:
maybe FieldValue.serverTimestamp() ?

Reply:
Aaaah! There is a topic about all dates inside this project which use `java.util.Date` but I didn't know that we can use this too to initialize dates everywhere. I'll make the change!